engine/src/main/java/org/terasology/engine/logic/players/UpdateDirectionEvent.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.engine.logic.players;
+
+import org.terasology.engine.entitySystem.event.Event;
+import org.terasology.engine.network.OwnerEvent;
+
+@OwnerEvent
+public class UpdateDirectionEvent implements Event {
```

Comment:
This event should have some docstring explaining what it is used for and how to use it. Try to categorize it with the event types we mentioned before.

Reply:
As mentioned in my other comment, maybe we should rather call this `SetDirectionEvent`?